src/java/org/apache/cassandra/index/sai/StorageAttachedIndex.java

```diff
@@ -40,16 +41,21 @@
 import java.util.stream.Collectors;
 
 import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
 import com.google.common.collect.ImmutableSet;
 import com.google.common.util.concurrent.Futures; // checkstyle: permit this import
 import com.google.common.util.concurrent.ListenableFuture; // checkstyle: permit this import
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import io.github.jbellis.jvector.vector.VectorSimilarityFunction;
```

Comment:
CQL similarity functions (`org.apache.cassandra.cql3.functions.VectorFcts` and `org.apache.cassandra.cql3.functions.VectorFctsTest`) still import `org.apache.lucene.index.VectorSimilarityFunction`. They should be updated to import `io.github.jbellis.jvector.vector.VectorSimilarityFunction`. They also contain some vars named `luceneFunction` that should be renamed.

Reply:
Thanks, I thought I had caught all of those. I will do a final check before putting this up for review again that I have caught all of these.

Reply:
There is still a Lucene import on `VectorFctsTest`.